apbs/chemistry/atom_list.py

```diff
@@ -34,13 +37,14 @@ def center(self) -> Coordinate:
             ma = self.maxcrd()
             mi = self.mincrd()
             self._dp["center"] = Coordinate(
-                (ma.x + mi.x) * 0.5, (ma.y + mi.y) * 0.5, (ma.z + mi.z) * 0.5,
+                (ma.x + mi.x) * 0.5,
+                (ma.y + mi.y) * 0.5,
+                (ma.z + mi.z) * 0.5,
             )
         return self._dp["center"]
 
     def mincrd(self) -> Coordinate:
-        """Minimum coordinates
-        """
+        """Minimum coordinates"""
```

Comment:
FWIW, I would have named this function (and its "maxcrd" equivalent) something like `max_coord` to make the code a bit more readable.

Reply:
Fixed